### DISCUSSION TITLE
Change of favicon was successful, but not as bookmark

### DISCUSSION THREAD

**Discussion by sarahwarchhold** [alshedivat/al-folio#2621] [Q&A] [answered]
I successfully changed the favicon in the _config.yml, but when I save my page as bookmark (in firefox) it still shows the old favicon next to my name. Any ideas what I need to adjust to show the new one there as well?
Thx a lot!

**Comment by george-gca** [comment]
Sometimes your browser just need to refresh the icon. Try deleting it from favorite and adding again.

**Reply by sarahwarchhold** [reply]
I tried, but no luck (also clearing cache, restarting everything)

**Reply by george-gca** [reply]
Googling around this seems to be a very common issue for firefox. I suggest trying some of the solutions suggested [here](https://stackoverflow.com/questions/37859753/refresh-favicon-in-bookmarks-firefox). I am afraid there's not much we can do on our side.

**Reply by sarahwarchhold** [reply]
Will do, thank you!

**Comment by sarahwarchhold** [comment, marked as answer]
Method 3 fixed it for me (force firefox to update icons): https://thegeekpage.com/how-to-fix-firefox-browser-not-showing-wrong-bookmark-favicons/
